runbooks/how-to-create-server-regression-dashboards.md

```diff
@@ -0,0 +1,343 @@
+       1. !["Format visual" Pane](resources/format-visual-pane.png)
+   22. There should be multiple collapsible sections, click on "Data labels"
+   23. Click on the "Show data labels" checkbox. Each point on the line graph should now be labeled with its value
+   24. Hover over the graph again and this time click the three-dots in the top right corner. A dropdown menu should appear
+   25. Click "Duplicate visual"
+   26. A duplicate of the original "Total RPS Over Time" line graph should appear to the right
+   27. Click on this new, duplicate graph. It should become outlined
+   28. Uncollapse the "Field wells" section above the graphs by clicking anywhere within it
+   29. Click on the green "total_reqs_per_second (Sum)" button in the "Value" section. A dropdown menu should appear
+   30. In the very bottom of the dropdown menu, beneath the "Search fields" input, find "total_requests". Click on it
+   31. The graph's y-axis should change to "total_requests (Sum)" and the "Value" of the graph should now be "total_requests (Sum)"
+   32. Change the title of the graph to "Total Requests over Time" following instructions above
+   33. Change the title of the y-axis to "Total Requests"
+   34. Ensure the title of the x-axis remains "Date and Time (EST)". If not, change it following instructions above
+   35. Ensure that data labels appear above each point in the graph. If not, change it following instructions above
+   36. Duplicate any of the two graphs following the instructions above. A third graph should appear on bottom-left of the screen
+   37. Following instructions outlined above, change the "Value"/y-axis of the graph to "total_failures"
+   38. Change the title to "Number of Total Request Failures over Time"
+   39. Change the y-axis title to "Total Failures"
+   40. Ensure the x-axis title is "Date and Time (EST)"
+   41. Ensure data labels appear above each point in the line graph
+   42. Navigate to "Format visual" (ensure that the new failures graph is selected)
+   43. In the "Title" collapsible section, click the "Edit subtitle" button
+   44. Change the subtitle to "Successful runs should always have 0 failures"
+
+9. Next, recreate the totals bar charts
+
+   1. On the top-left of the page, click the "ADD" button. A dropdown menu should appear
+   2. Select "Add visual". A new "AutoGraph" should appear on the page
+   3. From "Visual types" (bottom collapsible section of the left-pane) select "Vertical bar chart" (top-most row, right-most icon)
+   4. Click the "ADD" button again. Choose "Add calculated field". A new page should open with three main sections: a name field with the value "Add name", a section on the right with inner collapsible sections showing "Fields", "Parameters" and "Functions", and a large text area that takes up most of the screen
+   5. In the large text area, type "1"
+   6. For the name of the field, enter "fake_x". See the figure below:
```

Comment:
I agree with that, without context these steps definitely seem arbitrary or even incorrect. 

There is a good reason for them, though, don't worry! Specifically, we're working around QuickSight's requirement for vertical bar charts to have an x-axis to plot on, even though we're only really using the vertical bar chart to visualize various aggregations (min, median, average, max) of a particular metric together. This `fake_x` calculated field lets us use a vertical bar chart to do this visualization by plotting them against a single, _fake_ x-axis value: `1`. Later steps hide the x-axis from the chart so that this trick doesn't cause confusion for viewers.